src/Plugin.php

```diff
@@ -10,14 +10,14 @@
 class Plugin implements PluginInterface, Capable
 {
     /**
-     * {@inheritDoc}
+     * @return void
```

Comment:
since they all come from the parent interface maybe we can just remove them (the phpdoc) altogether?

Reply:
Unfortunately that doesn't fix the triggered deprecation. Only explicit helps :(

Reply:
Too bad